packages/test/test-service-load/testConfig.json

```diff
@@ -144,6 +144,26 @@
                 "min": 900000,
                 "max": 1800000
             }
+        },
+        "scale_with_signals_concurrent":  {
+            "opRatePerMin":  7,
+            "signalsPerMin":  46,
+            "totalSignalsSendCount":  420000,
+            "progressIntervalMs":  20000,
+            "numClients":  10,
+            "totalSendCount":  70000,
+            "readWriteCycleMs":  30000,
+            "faultInjectionMinMs":  900000,
```

Comment:
is it a problem that this is not the same format with the above profiles? like is it supposed to be
` "faultInjectionMs": {
                "min": 900000,
                "max": 1800000
            }`

Reply:
nice catch! Yeah, looks like it is based on call site in `runner.ts`. Right now this config won't cause fault injection.

Reply:
https://github.com/microsoft/FluidFramework/pull/13357